.github/workflows/gradle-publish.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Publish package to the Maven Central Repository with Gradle
+
+env:
+  nexusUsername: ${{ secrets.NEXUS_USER }}
+  nexusPassword: ${{ secrets.NEXUS_PASSWORD }}
+  PGP_SIGNING_KEY: ${{ secrets.PGP_SIGNING_KEY }}
+  PGP_SIGNING_PASSWORD: ${{ secrets.PGP_SIGNING_PASSWORD }}
+
+on:
+  release:
+    types: [ created ]
```

Comment:
Minor detail: do we want to do it on `created` or `published` event?

Reply:
We might want to do it on `created` event if we want to create a draft release in GitHub and test the release published to Maven. I guess the question is then: can we remove it from Maven if we don't like it? If not, then this use-case is probably irrelevant, and we just want to trigger the release process on publishing.
What do you think?